src/js/edu-benefits/1990e/config/form.js

```diff
@@ -6,8 +6,9 @@ import {
 import fullSchema1990e from 'vets-json-schema/dist/transfer-benefits-schema.json';
 
 import * as currentOrPastDate from '../../../common/schemaform/definitions/currentOrPastDate';
-import * as fullName from '../../../common/schemaform/definitions/fullName';
-import * as ssn from '../../../common/schemaform/definitions/ssn';
+import * as fullNameCommon from '../../../common/schemaform/definitions/fullName';
```

Comment:
@annekainicUSDS I think we're on the same page as far as what schema you should be using (what you're doing now is what I think we agreed on), Raquel's comment was more just about how to import the uiSchema for fullName. You can use the import statement in my comment to replace `fullNameCommon.uiSchema` with `fullNameUiSchema`, which I think is clearer and it makes it easier to remove the common schema from that file later.